base/compiler/ssair/passes.jl

```diff
@@ -1311,13 +1395,14 @@ function cfg_simplify!(ir::IRCode)
             bb_rename_pred[i] = -1
             continue
         end
-        bbnum = follow_merge_into(i)
+        pred = i
+        while pred !== 1 && !Core.Main.Base.isempty(Core.Main.Base.searchsorted(dropped_bbs, pred))
```

Comment:
`Base` seems forbidden here. How does this not crash?

Reply:
Whoops. That was a debugging thing that got left in. Will fix. The reason it doesn't show up on CI is that this isn't part of our pass pipeline so it doesn't get run during bootstrap (and Base is available in the tests).